assessment.sql

```diff
@@ -5,24 +5,58 @@ CREATE DATABASE "sql-crud-assessment";
 
 -- Question 1. Create a table called `licensees` to hold data in columns surname,given_name,gender,height,weight,dob,eye_color,hair_color.  See `data/licensees.csv` for example data.  Don't forget that all tables should have an id column as a unique identifier for a row.
 
+CREATE TABLE licencees(
+  P_Id int NOT NULL,
+  surname CHARACTER VARYING(128),
+  given_name CHARACTER VARYING(128),
+  gender CHARACTER VARYING(1),
+  height INTEGER,
+  weight INTEGER,
+  dob DATE,
+  eye_color CHARACTER VARYING (10),
+  hair_color CHARACTER VARYING (10),
+  PRIMARY KEY (P_Id)
+);
 -- your code here
 
 -- Question 2. Add `Bishop, Caren, f, 63, 132, 1943-09-26, Brown, Black` into the table of licensees.
 
+INSERT INTO licensees(surname,given_name,gender,height,weight,dob,eye_color,hair_color) VALUES ('Bishop', 'Caren', 'f', 63, 132, '1943-09-26', 'Brown', 'Black');
 -- your code here
 
+
 -- Question 3. Load the data from `data/licensees.csv` into the `licensees` table.
 
+\copy licensees(surname,given_name,gender,height,weight,dob,eye_color,hair_color) FROM 'data/licensees.csv' WITH (HEADER true, FORMAT csv);
 -- your code here
 
+
 -- Question 4. Write a query to get all attributes of licensees with Hazel eye color and Brown or Black hair color.
 
+SELECT count(*), eye_color,hair_color
+FROM licensees
+WHERE eye_color = 'Hazel'
```

Comment:
Close. The `AND` is done first because it takes precedence over `OR`, so this WHERE clause actually looks for people with both hazel eyes and brown hair, or just black hair.